lib/collection/src/recommendations.rs

```diff
@@ -36,10 +36,10 @@ fn avg_vectors<'a>(
 ) -> CollectionResult<VectorInternal> {
     let mut avg_dense = DenseVector::default();
     let mut avg_sparse = SparseVector::default();
-    let mut avg_multi: Option<TypedMultiDenseVector<VectorElementType>> = None;
+    let mut avg_multi: Option<DenseVector> = None;
```

Comment:
I think we want to keep the existing behavior for single multivector.
That is not average it at all.

Reply:
Ah, I see, I forgot we don't do `pos + pos - neg` if there are no negatives.
I will change it so that single multivector is just passed as-is

Reply:
https://github.com/qdrant/qdrant/pull/6308/commits/029a30b039eedb65c71a98a86d4b4365943452a7

Reply:
I'd love to have a short unit test showing the new strategy in action.
One positive example, one negative example etc... :)

Reply:
in https://github.com/qdrant/qdrant/pull/6308/commits/0aa7f77ef74b55b1c27d98df9c38dafae25547da I added a test to make sure that the averaging is the same as for dense, but flattened